data/pokedex.js

```diff
@@ -430,6 +432,7 @@ let BattlePokedex = {
 		weightkg: 105,
 		color: "Black",
 		eggGroups: ["Field"],
+		hide: "SpriteOnly",
```

Comment:
I believe @Marty-D said the totems do have competitive differences beyond their sprite, such as weight (which matters for doubles?)

Reply:
i dont think any of the totems are used competitively in smogon formats at least (except for a small period in time where totem araquanid was viable in RU)